packages/node-terra/src/utils/terra-helper.ts

```diff
@@ -40,7 +41,11 @@ export function filterMessageData(
   if (
     filter.type === '/terra.wasm.v1beta1.MsgExecuteContract' &&
     filter.contractCall &&
-    !(filter.contractCall in (data.msg as MsgExecuteContract).execute_msg)
+    typeof (data.msg as MsgExecuteContract).execute_msg === 'object' &&
```

Comment:
I think if its type string then we should do `execute_msg.contains(filter.ContractCall)`